dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -474,8 +474,12 @@ private void updateDBWorkflowWithSourceControlWorkflow(Workflow workflow, Workfl
 
         // Then copy over content that changed
         for (WorkflowVersion version : newWorkflow.getWorkflowVersions()) {
+            // skip frozen versions
             WorkflowVersion workflowVersionFromDB = existingVersionMap.get(version.getName());
             if (existingVersionMap.containsKey(version.getName())) {
+                if (workflowVersionFromDB.isFrozen()) {
```

Comment:
You could alternatively change line 476 to use stream/filter and not check here.

But that would be a bigger code change, so maybe better to keep it focused for this PR.